lua_examples/pcm/play_file.lua

```diff
@@ -0,0 +1,40 @@
+-- ****************************************************************************
+-- Play file with pcm module.
+--
+-- Upload jump_8k.u8 to spiffs before running this script.
+--
+-- ****************************************************************************
+
+
+function cb_drained(d)
+  print("drained "..node.heap())
+
+  file.seek("set", 0)
+  -- uncomment the following line for continuous playback
+  --d:play(pcm.RATE_8K)
+end
+
+function cb_stopped(d)
+  print("playback stopped")
+  file.seek("set", 0)
+end
+
+function cb_paused(d)
+  print("playback paused")
+end
+
+file.open("jump_8k.u8", "r")
+
+drv = pcm.new(pcm.SD, 1)
```

Comment:
No, I can just guess that the user had an old state of my `pcm` branch when he tried initially. `pcm.SD` was introduced around 12-Apr, not sure when this change finally became visible on github.
Running pcm while having the sigma-delta module disabled definitely works for me. I tried it right now to be sure :sweat_smile: